.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,81 @@
+name: Container for Kanidm
+
+# this will build regardless, but only push to the container registry when you're committing on the master branch.
```

Comment:
I don't think we should always build this, I think the CI tests are enough, we shoulrd only build on master

Reply:
I just want to add here that this can help to catch up bugs in the container build step.
It's important if you want to keep stability in your releases.
In my opinion tests are free so any little help should be appreciated.

Reply:
@pando85's point matches my opinion - if it's free and catches issues in the build. 🤷🏻

Reply:
Yep fair enough.